specification/README.md

```diff
@@ -0,0 +1,22 @@
+Jakarta Concurrency Specification
+============================
+
+This project generates the Jakarta Concurrency 1.1 Specification.
```

Comment:
Should be 1.0 rather than 1.1 to match the Java EE spec version

Reply:
Changes made